### ISSUE TITLE
Add function to add distance and time to daily logs

### ISSUE THREAD

**Issue by jait** [Tumetsu/kilometrikisa-client#20]
It would be convenient if there was a way to log more kilometers for a day, not just set the daily total.

Input: date,  kilometers to add, e-bike or not

If the API does not allow incrementing the daily amount, the function could simply fetch the distance for the day, add the amount to it and save it back as the distance for the day. Yes, the logic has a potential race condition but this would be still much better than nothing.

Similar thing would be nice for the riding times.

**Comment by Tumetsu** [comment]
Yes that would be convenient. The Kilometrikisa api allows only setting kilometers and minutes for specific date. However, it should be possible to first get the current amount of distance and minutes by using `getUserContestLogEntries` api. Now when Kilometrikisa has actually started it is possible to test if that actually works properly and perhaps develop a new api for adding kilometers instead of just setting them.
